adobe_vipm/flows/fulfillment/renewal_now.py

```
@@ -250,6 +281,173 @@ def _fail_order(self, client, context, order, error):
+    def _get_completed_renewal_orders(self, adobe_client, context):
+        orders = adobe_client.get_orders(
+            context.authorization_id,
+            context.adobe_customer_id,
+            filters={
+                "order-type": ORDER_TYPE_RENEWAL,
+                "status": AdobeOrderStatus.COMPLETE,
+            },
+        )
+        return sorted(orders, key=itemgetter("creationDate"), reverse=True)
+
+    def _return_previous_renewal(  # noqa: WPS211
```

Ruff (0.16.1)

[warning] Invalid rule code in `# noqa`: WPS211

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)